readthedocs/templates/builds/build_detail.html

```diff
@@ -62,7 +63,11 @@
 
     <div class="build-version">
       <span class="build-version-slug">
-        {{ build.version.slug }}
+        {% if request.user|is_admin:project %}
+          <a href="{% url "project_version_detail" build.version.project.slug build.version.slug %}">{{ build.version.slug }}</a>
```

Comment:
This should probably add an "Edit this version" link, not just link the actual version name.

Reply:
Yeah, I didn't know where/how to put to not break the style since the commit hash is already between parenthesis (and I like that) so, I was thinking on adding the "Edit this version" in the following line. What do you think?

Reply:
Example

```
0.14.4 (acbd8asd67a8761asvsd)
Edit this version
```

Reply:
I'd prefer the text link actually. "Edit this version" is a verbose addition. I think working a button into the interface is adding another design pattern to track. I'd rather revisit this when we have a better idea of what design patterns we're using on these interfaces.